StarryExpanse/Structs/InteractabilityPollResponse.h

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2017 59 Volt Entertainment, all rights reserved.
+//
+#pragma once
+
+#include "Actors/EInteractable.h"
+#include "CoreMinimal.h"
+#include "InteractabilityPollResponse.generated.h"
+
+USTRUCT(BlueprintType)
+struct FInteractabilityPollResponse {
+  GENERATED_BODY()
+
+  static FInteractabilityPollResponse BasicResponse();
+
+  // Truth here indicates that this struct is not necessarily accurate
+  // at the actor level, and a poll should be performed using the actual components.
+  // Currently not implemented, but might be nice for Artist Engine Specialists. TODO(philip)
+  UPROPERTY(BlueprintReadWrite)
+  bool ShouldScanAtComponentLevel;
+
+  UPROPERTY(BlueprintReadWrite)
+  bool CanBeDragged;
+
+  // -1 = zooming out;
+  //  0 = not zooming;
+  // +1 = zooming in
+  UPROPERTY(BlueprintReadWrite)
+  int Zoomyness;
```

Comment:
How about just calling this ZoomState?

Reply:
Well it's more of a cue. Maybe ZoomCue? I just liked the word "zoomyness" and felt zany :P

Reply:
I like zany - let's now breed it out of the product. It's just that when I read the word it reads (to my eye) like the various states of zoom that it can be in instead of the one state that it currently is (or wants to be) in.